### DISCUSSION TITLE
weekly/2024-06-30

### DISCUSSION THREAD

**Discussion by giscus** [ATQQ/sugar-blog#262] [Announcements]
# weekly/2024-06-30

① 豆包 MarsCode 正式发布 ② ECMAScript 2024 正式发布 ③ Mako 开源 - 蚂蚁的 Rust 力作 ④ CSDN批量搬运Github项目伪造开发者主页 ⑤ HTML 旋转图像实现示例 ⑥ 一组看着糙的组件库 ⑦ Chrome 126 中 DevTools 的新增功能 ⑧ 纯 CSS 实现环形文本

https://sugarat.top/weekly/2024-06-30.html

**Comment by mumingfang** [comment]
您好，请问这个类似周报或者周刊的有专门的分类么？我们是一个网址导航，有个专门收集期刊的板块，想收录贵站，但是没找到一个分类列表 https://n.mumingfang.com/my#term-98

**Reply by ATQQ** [reply]
目前确实没做单独的周刊页，如果需要我可以后面定制一下，你可以先使用这个地址，我在后面调整一下排版和视图：https://sugarat.top/weekly/index.html

现在只有单独的RSS订阅地址：https://sugarat.top/weekly.rss
